analyses/copy_number_consensus_call/src/scripts/remove_dup_NULL_entries.py

```diff
@@ -0,0 +1,50 @@
+################## PURPOSE #################
+# This script is to remove duplicated coordinates reulted from Bedtools merging
+# and removes NULLs if they are not stand-alone values in a cell
+
+################# ASSUMPTION ###############
+
+
+############################################
+
+# Imports in the pep8 order https://www.python.org/dev/peps/pep-0008/#imports
+# Standard library
+import argparse
+import sys
+
+# Related third party
+import numpy as np
+import pandas as pd
+
+
+def remove_dup_and_null(cell):
+    """ Remove duplicated coordinates and NULLs """
+
+    ## Split the cell by commas and convert it to a set, this removes duplicated elements
+    set_cell = set(cell.split(','))
+
+    ## Keep NULLs if it's the only entry in a cell, else, remove NULLs
+    if "NULL" in set_cell and len(set_cell) > 1:
+        set_cell.remove("NULL")
+
+    ## Join the set
+    new_cell = ",".join(set_cell)
```

Comment:
While we are at it, lets make sure these are sorted by the start position:
```suggestion
    sorted_cell = sorted(set_cell, key=lambda x: x.split(":")[0])
    new_cell = ",".join(sorted_cell)
```